.github/workflows/build-website.yml

```diff
@@ -1,75 +1,91 @@
-# Copyright (C) 2020  Matthew Glazar
-# See end of file for extended copyright information.
-
 name: build website
-on:
+on: 
   push:
   pull_request:
     types: [opened, synchronize]
 
+
 jobs:
   build:
     name: website
     runs-on: ubuntu-latest
-    # TODO(strager): Use an image based on
-    # quick-lint-js-web.
-    container: ghcr.io/quick-lint/quick-lint-js-github-builder:v3
+    container: ghcr.io/k1ng440/quick-lint-js-website-builder:v1
     env:
       QLJS_COLLECT_COPYRIGHT_NO_WARNINGS: 1
+      EM_VERSION: 2.0.4
+      EM_CACHE_FOLDER: 'emsdk-cache'
+
     steps:
-      - uses: quick-lint/setup-emsdk@v7
-        with:
-          version: 2.0.4
       - name: checkout
         uses: actions/checkout@v2
 
+      - name: Setup emsdk cache
+        id: cache-system-libraries
+        uses: actions/cache@v2
+        with:
+          path: ${{env.EM_CACHE_FOLDER}}
+          key: ${{env.EM_VERSION}}-${{ runner.os }}
+
+      - uses: quick-lint/setup-emsdk@v7
+        with:
+          version: ${{env.EM_VERSION}}
+          actions-cache-folder: ${{env.EM_CACHE_FOLDER}}
```

Comment:
What does this have to do with switching to a Docker container?

Reply:
this pull request is a kind of complete overhaul of this workflow.